wasmtime/_module.py

```diff
@@ -1,14 +1,15 @@
 from ._ffi import *
 from ctypes import *
 from wasmtime import Store, wat2wasm, ImportType, ExportType, WasmtimeError
+import typing
 
 dll.wasmtime_module_new.restype = P_wasmtime_error_t
 dll.wasmtime_module_validate.restype = P_wasmtime_error_t
 
 
 class Module:
     @classmethod
-    def from_file(cls, store, path):
+    def from_file(cls, store: Store, path):
```

Comment:
I think `path` here is missing a type annotation?

Reply:
(is there a way to catch this?)

Reply:
I've not tried to annotate every type, actually. Trying to do that with Python, you'll end up hitting a point of diminishing returns. `path` can be things other than a `str` here but I wasn't sure what to do. `str` would be okay.